ui/src/shared/components/RefreshingGraph.js

```diff
@@ -51,6 +52,7 @@ const RefreshingGraph = ({
 
```

Comment:
The update key should also be passed to `RefreshingSingleStat`.

Reply:
Good catch!!!